float64[] goal_vector
---
bool is_successful
